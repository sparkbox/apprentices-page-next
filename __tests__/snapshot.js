import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../pages/index';

it('renders homepage unchanged', () => {
  const apprenticeData = {
    currentApprenticeGroup: {
      version: '0.0',
      currentApprentices: [
        {
          name: 'First Last',
          image: '/apprentices/First.png',
          links: [
            { href: 'personal', text: 'Personal' },
            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
            { href: 'https://www.github.com/', text: 'Github' },
          ],
        },
        {
          name: 'First Last',
          image: '/apprentices/First.png',
          links: [
            { href: 'personal', text: 'Personal' },
            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
            { href: 'https://www.github.com/', text: 'Github' },
          ],
        },
      ],
    },
    previousApprenticeGroups: [{
      version: '0.0',
      apprentices: [
        {
          name: 'First Last',
        },
        {
          name: 'First Last',
          status: 'previous',
        },
        {
          name: 'First Last',
          status: 'current',
        },
      ],
    }],
  };
  const tree = renderer.create(<Index apprenticeData={apprenticeData} />).toJSON();
  expect(tree).toMatchSnapshot();
});
