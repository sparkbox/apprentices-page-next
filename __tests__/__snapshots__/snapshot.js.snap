// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="container"
>
  <main>
    <header
      className="hero"
    >
      <div
        className="hero__container"
      >
        <div>
          <svg
            aria-hidden={true}
            className="hero__sparkbox-logo--half"
            fill="none"
            height="45"
            width="41"
            xmlns="http://www.w3.org/2000/svg"
          >
            <span>
              Sparkbox
            </span>
            <path
              d="M9.84 29.708v6.155h1.968l3.635-6.155H9.84Zm20.883-13.35h.027a3.087 3.087 0 0 0 3.089-3.09 3.087 3.087 0 0 0-3.089-3.092h-.793l-3.635 6.155 4.4.028ZM9.84 10.205v7.003l10.387-7.003H9.84Zm20.91 19.504h-1.011l-9.13 6.155H30.75a3.087 3.087 0 0 0 3.089-3.091c0-1.669-1.394-3.064-3.089-3.064Zm9.922-7.358-7.68 5.197a5.706 5.706 0 0 1 3.47 5.252 5.712 5.712 0 0 1-5.712 5.718H16.728l-9.54 6.428c-.054.028-.109.055-.19.055a.316.316 0 0 1-.329-.328c0-.082.028-.137.055-.192l3.526-5.99H7.517a.333.333 0 0 1-.328-.329v-10.75c0-.192.136-.329.328-.329h11.917c.164 0 .328.137.328.328a.395.395 0 0 1-.055.192l-5.576 9.465 18.86-12.748H.328A.316.316 0 0 1 0 23.991c0-.11.055-.219.164-.274l7.052-4.76V7.851c0-.164.164-.3.328-.3h16.564l7.243-4.897 3.854-2.6A.395.395 0 0 1 35.397 0c.191 0 .328.137.328.328a.508.508 0 0 1-.055.192l-4.155 7.085a5.703 5.703 0 0 1 4.975 5.662 5.712 5.712 0 0 1-5.713 5.718h-8.446a.316.316 0 0 1-.328-.328c0-.055.028-.11.028-.165l6.204-10.559L7.845 21.72h32.69c.192 0 .328.137.328.329-.027.136-.082.246-.191.3Z"
              fill="#fff"
            />
          </svg>
          <a
            href="https://www.seesparkbox.com"
          >
            <span
              className="hero__header-label"
            >
              Sparkbox
            </span>
            <svg
              aria-hidden={true}
              className="hero__sparkbox-logo--full"
              fill="none"
              height="52"
              width="193"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11.225 34.33v7.112h2.245l4.146-7.112h-6.391Zm23.82-15.427h.032c1.933 0 3.523-1.58 3.523-3.572 0-1.96-1.559-3.572-3.523-3.572h-.904l-4.147 7.113 5.02.031Zm-23.82-7.112v8.092l11.848-8.092H11.225ZM35.077 34.33h-1.154L23.51 41.442h11.568c1.933 0 3.523-1.58 3.523-3.572 0-1.928-1.59-3.54-3.523-3.54Zm11.318-8.504-8.762 6.006a6.596 6.596 0 0 1 3.96 6.07c0 3.635-2.9 6.606-6.516 6.606H19.082L8.2 51.937a.446.446 0 0 1-.218.063.363.363 0 0 1-.374-.38c0-.094.03-.157.062-.22l4.022-6.923H8.574a.382.382 0 0 1-.374-.38V31.674c0-.221.156-.38.374-.38H22.17c.187 0 .374.159.374.38a.461.461 0 0 1-.063.221l-6.36 10.938 21.513-14.73H.374a.363.363 0 0 1-.374-.38c0-.127.062-.253.187-.316l8.044-5.5V9.071c0-.19.187-.347.374-.347H27.5l8.263-5.659L40.159.063A.446.446 0 0 1 40.377 0c.218 0 .374.158.374.38a.592.592 0 0 1-.062.22l-4.74 8.188c3.212.41 5.675 3.192 5.675 6.543 0 3.636-2.9 6.607-6.516 6.607h-9.634a.363.363 0 0 1-.375-.38c0-.063.032-.126.032-.189l7.077-12.202L8.948 25.1H46.24c.218 0 .374.158.374.38-.031.158-.093.284-.218.347ZM72.648 19.314l-2.65 1.454c-.406-.885-1.092-2.75-3.274-2.655-1.216.032-2.339 1.233-2.339 2.94 0 1.738.749 2.94 3.15 4.394 3.21 1.991 4.832 4.14 4.832 7.65 0 3.983-2.245 6.764-5.955 6.764-3.742 0-5.425-2.529-6.454-5.563l2.93-1.454c.656 1.643 1.31 3.572 3.15 3.603 1.528.032 2.744-1.106 2.744-2.971 0-1.707-.686-2.94-3.368-4.52-2.837-1.707-4.614-3.983-4.614-7.524 0-3.825 2.65-6.733 5.8-6.733 4.302 0 5.58 3.572 6.048 4.615ZM77.45 15.047h4.707c4.303 0 6.922 3.034 6.922 7.997 0 5.027-2.619 8.125-6.86 8.125h-1.122v8.187H77.45v-24.31Zm3.647 12.644h.842c2.12 0 3.337-1.707 3.337-4.615 0-2.845-1.248-4.52-3.368-4.52h-.81v9.135ZM101.208 33.919H95.97l-1.091 5.468h-3.523l5.269-24.34h4.115l5.207 24.34h-3.648l-1.091-5.468Zm-4.614-3.32h3.991l-1.934-9.704h-.093l-1.964 9.704ZM110.406 15.047h4.427c4.397 0 7.078 2.908 7.078 7.587 0 2.908-.904 5.12-2.557 6.511l3.586 10.242h-3.928l-2.931-8.85c-.468.063-.905.126-1.372.126h-.624v8.724h-3.648v-24.34h-.031Zm3.679 12.17h.561c2.214 0 3.492-1.58 3.492-4.489 0-2.718-1.278-4.172-3.492-4.172h-.561v8.661ZM133.354 29.525l-1.621 3.034v6.797h-3.648v-24.34h3.648V26.71l4.708-11.696h4.053l-4.739 10.653 5.238 13.688h-4.178l-3.461-9.831ZM145.732 15.047h5.457c3.71 0 6.204 2.529 6.204 6.322 0 2.402-1.091 4.489-2.806 5.374 1.933.822 3.212 2.971 3.212 5.627 0 4.204-2.682 6.986-6.61 6.986h-5.426l-.031-24.31Zm3.586 10.463h1.559c1.652 0 2.712-1.454 2.712-3.762 0-2.023-1.06-3.414-2.681-3.414h-1.59v7.176Zm1.621 10.621c1.84 0 3.025-1.517 3.025-3.761 0-2.181-1.185-3.604-3.056-3.604h-1.59v7.365h1.621ZM168.992 39.798c-4.615 0-7.203-4.489-7.203-12.613 0-8.092 2.62-12.55 7.203-12.55 4.614 0 7.202 4.521 7.202 12.614 0 8.092-2.588 12.549-7.202 12.549Zm0-3.667c2.12 0 3.336-3.16 3.336-8.914 0-5.722-1.247-8.883-3.336-8.883-2.151 0-3.336 3.161-3.336 8.883 0 5.753 1.185 8.914 3.336 8.914ZM183.865 27.09l-4.709-12.043h4.147l2.993 8.693 2.775-8.693h3.835l-4.614 12.296L193 39.388h-4.116l-3.024-8.724-2.931 8.724h-3.804l4.74-12.296Z"
                fill="#fff"
              />
            </svg>
          </a>
        </div>
        <div
          className="hero__header-content"
        >
          <div
            className="hero__text"
          >
            <h1>
              Apprenticeships at Sparkbox
            </h1>
            <p>
              Sparkbox is a team that values education. We offer paid apprenticeships because we love what we do, we want to share with new talent, and we genuinely care about the future of the web.
            </p>
          </div>
          <div
            aria-hidden={true}
            className="hero__art-container"
          >
            <div
              className="hero__grid"
            >
              <img
                alt=""
                src="/apprentices/Luis.png"
              />
              <img
                alt=""
                src="/apprentices/Rise.png"
              />
              <img
                alt=""
                src="/apprentices/Melissa.png"
              />
              <img
                alt=""
                src="/apprentices/Flash.png"
              />
              <img
                alt=""
                src="/apprentices/Susmita.png"
              />
              <div
                className="hero__grid--shape1"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 34"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                    stroke="#FE3CB0"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape2"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 40"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="19.9315"
                    cy="19.7905"
                    r="18.2325"
                    stroke="#01F5AC"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape3"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 34"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                    stroke="#FE3CB0"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape4"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 40"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="19.9315"
                    cy="19.7905"
                    r="18.2325"
                    stroke="#01F5AC"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape5"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 34"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                    stroke="#FE3CB0"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape6"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 40"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="19.9315"
                    cy="19.7905"
                    r="18.2325"
                    stroke="#01F5AC"
                  />
                </svg>
              </div>
              <div
                className="hero__grid--shape7"
              >
                <svg
                  fill="none"
                  viewBox="0 0 40 34"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                    stroke="#FE3CB0"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
    </header>
    <section
      className="apprentice-qualities"
    >
      <div
        className="apprentice-qualities__main"
      >
        <img
          alt="The three qualities sparkbox looks for in apprentices: Passion, Character, and Initiative."
          className="apprentice-qualities__image"
          src="/apprentice-qualities.svg"
        />
        <article
          className="apprentice-qualities__article"
        >
          <p
            className="primary apprentice-qualities__article-heading"
          >
            What we look for
          </p>
          <h2
            className="apprentice-qualities__article-subheading"
          >
            Qualities of Successful Apprentices
          </h2>
          <div
            className="apprentice-qualities__article-text"
          >
            <p>
              We build responsive, standards-compliant, content-focused, well-crafted, accessible, valuable, beautiful web sites and apps, and we're looking for some web geeks that want to do the same.
            </p>
            <p>
              Passion, character, and initiative are a requirement. A long resumé is not—as long as you care about the right things, we can help build your skillset. This is true for those we hire, and it is equally true for our apprentices. Through your apprenticeship, we hope to help develop you into the sort of individual we’d want to hire. And we may do just that.
            </p>
            <p>
              We’re looking primarily for those interested in an apprenticeship in web development—frontend, backend, or anything code related. What we’re looking for most, however, are individuals who are excited to make their mark on the web industry. In your application, provide anything and everything you can that will convince us you are that geek.
            </p>
          </div>
        </article>
      </div>
    </section>
    <section
      className="current-apprentices"
    >
      <div
        className="current-apprentices__content"
      >
        <div
          className="current-apprentices__header"
        >
          <h2>
            Current Apprentices
            <span
              className="current-apprentices__version"
            >
               
              v
              0.0
            </span>
          </h2>
        </div>
        <div
          className="card-container"
        >
          <div
            className="card-container__card"
          >
            <article
              className="container"
            >
              <h2
                className="container__heading"
              >
                First Last
              </h2>
              <img
                alt="First Last"
                className="container__image"
                src="/apprentices/First.png"
              />
              <div
                className="container__links"
              >
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="personal"
                  >
                    Personal
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="https://www.linkedin.com/"
                  >
                    Linkedin
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="https://www.github.com/"
                  >
                    Github
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
              </div>
            </article>
          </div>
          <div
            className="card-container__card"
          >
            <article
              className="container"
            >
              <h2
                className="container__heading"
              >
                First Last
              </h2>
              <img
                alt="First Last"
                className="container__image"
                src="/apprentices/First.png"
              />
              <div
                className="container__links"
              >
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="personal"
                  >
                    Personal
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="https://www.linkedin.com/"
                  >
                    Linkedin
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  className="container"
                >
                  <a
                    className="link"
                    href="https://www.github.com/"
                  >
                    Github
                  </a>
                  <svg
                    fill="none"
                    height="9"
                    viewBox="0 0 11 9"
                    width="11"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M0.5 3.07343L0.5 8.5L1.26923 8.5L1.26923 3.07343L8.31903 3.07343C7.95887 3.47125 7.57203 4.06799 7.15851 4.86364L7.85883 4.86364C8.6992 3.94367 9.57959 3.27234 10.5 2.84965L10.5 2.51399C9.57959 2.07887 8.6992 1.40754 7.85883 0.500001L7.15852 0.500001C7.57203 1.29565 7.95887 1.89239 8.31903 2.29021L0.5 2.29021L0.5 2.68182L0.5 3.07343Z"
                      fill="white"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
              </div>
            </article>
          </div>
        </div>
      </div>
    </section>
    <article
      className="previous-apprentices"
    >
      <section
        className="previous-apprentices__header"
      >
        <h2>
          Previous Apprentices
        </h2>
        <div
          className="previous-apprentices__legend"
        >
          <div
            className="previous-apprentices__legend-item"
          >
            <svg
              className="employee-status"
              viewBox="0 0 100 100"
            >
              <polygon
                className="current-employee-status"
                points="50 15, 100 100, 0 100"
              />
            </svg>
            <p>
              Current Employee
            </p>
          </div>
          <div
            className="previous-apprentices__legend-item"
          >
            <svg
              className="employee-status"
              viewBox="0 0 100 100"
            >
              <polygon
                className="past-employee-status"
                points="50 15, 100 100, 0 100"
              />
            </svg>
            <p>
              Past Employee
            </p>
          </div>
        </div>
      </section>
      <div
        className="previous-apprentices__groups"
      >
        <section
          className="previous-apprentices-group"
        >
          <p
            className="previous-apprentices-group__version"
            data-testid="0.0"
          >
            0.0
          </p>
          <ul
            className="previous-apprentices-group__statuses"
          >
            <li
              className="previous-apprentices-group__apprentice"
              data-testid="First Last"
            >
              First Last
            </li>
            <li
              className="previous-apprentices-group__apprentice"
              data-testid="First Last"
            >
              First Last
              <svg
                className="employee-status"
                viewBox="0 0 100 100"
              >
                <polygon
                  className="past-employee-status"
                  data-testid="status-indicator"
                  points="50 15, 100 100, 0 100"
                />
              </svg>
            </li>
            <li
              className="previous-apprentices-group__apprentice"
              data-testid="First Last"
            >
              First Last
              <svg
                className="employee-status"
                viewBox="0 0 100 100"
              >
                <polygon
                  className="current-employee-status"
                  data-testid="status-indicator"
                  points="50 15, 100 100, 0 100"
                />
              </svg>
            </li>
          </ul>
        </section>
      </div>
    </article>
    <footer
      className="call-to-action"
    >
      <div
        className="call-to-action__container"
      >
        <div
          className="call-to-action__text-container"
        >
          <h2>
            Ready to Apply?
          </h2>
          <p>
            Take a dive into the world of clean, responsive, full-stack development as you learn what it takes to build a better web.
          </p>
          <a
            href="https://sparkbox.breezy.hr/p/a996600dfdcc-sparkbox-remote-full-stack-apprentice"
          >
            Learn More & Apply
          </a>
        </div>
        <div
          aria-hidden={true}
          className="call-to-action__art-container"
        >
          <div
            className="call-to-action__grid"
          >
            <img
              alt=""
              src="/apprentices/Michael.png"
            />
            <img
              alt=""
              src="/apprentices/Neil.png"
            />
            <img
              alt=""
              src="/apprentices/Sheridan.png"
            />
            <img
              alt=""
              src="/apprentices/Hunter.png"
            />
            <img
              alt=""
              src="/apprentices/Merani.png"
            />
            <div
              className="call-to-action__grid--shape1"
            >
              <svg
                fill="none"
                viewBox="0 0 40 34"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                  stroke="#FE3CB0"
                />
              </svg>
            </div>
            <div
              className="call-to-action__grid--shape2"
            >
              <svg
                fill="none"
                viewBox="0 0 40 40"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="19.9315"
                  cy="19.7905"
                  r="18.2325"
                  stroke="#01F5AC"
                />
              </svg>
            </div>
            <div
              className="call-to-action__grid--shape3"
            >
              <svg
                fill="none"
                viewBox="0 0 40 40"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="19.9315"
                  cy="19.7905"
                  r="18.2325"
                  stroke="#01F5AC"
                />
              </svg>
            </div>
            <div
              className="call-to-action__grid--shape4"
            >
              <svg
                fill="none"
                viewBox="0 0 40 34"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.00133 32.5401L20.2337 2.69278L37.4661 32.5401H3.00133Z"
                  stroke="#FE3CB0"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
      <div
        className="call-to-action__footer"
      >
        <p>
          This site was proudly built by Sparkbox apprentices Version 14.0
        </p>
        <a
          href="https://www.seesparkbox.com"
        >
          <span
            className="sr-only"
          >
            Sparkbox
          </span>
          <svg
            fill="none"
            height="33"
            width="29"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M6.96 22.058v4.377h1.392l2.571-4.377H6.96Zm14.77-9.493h.02a2.19 2.19 0 0 0 2.185-2.198A2.19 2.19 0 0 0 21.75 8.17h-.56l-2.572 4.377 3.113.02ZM6.96 8.19v4.98l7.347-4.98H6.96Zm14.79 13.87h-.715l-6.458 4.376h7.173a2.19 2.19 0 0 0 2.185-2.198 2.198 2.198 0 0 0-2.185-2.178Zm7.018-5.233-5.433 3.696a4.058 4.058 0 0 1 2.456 3.735c0 2.237-1.798 4.065-4.041 4.065h-9.918l-6.747 4.572a.278.278 0 0 1-.136.039.224.224 0 0 1-.232-.234c0-.058.02-.097.039-.136l2.494-4.26H5.317a.236.236 0 0 1-.232-.234v-7.645c0-.136.096-.233.232-.233h8.429c.116 0 .232.097.232.233 0 .059-.02.098-.039.137l-3.944 6.73 13.34-9.065H.232A.224.224 0 0 1 0 17.993c0-.078.039-.156.116-.195l4.988-3.384V6.515c0-.116.116-.213.232-.213h11.716l5.123-3.482L24.901.972c.039-.02.078-.04.136-.04.135 0 .232.098.232.234 0 .039-.02.097-.039.136l-2.939 5.039c1.992.252 3.519 1.964 3.519 4.026 0 2.237-1.798 4.066-4.04 4.066H15.794a.224.224 0 0 1-.232-.233c0-.04.02-.078.02-.117l4.388-7.51L5.55 16.379H28.67c.136 0 .232.098.232.234-.019.097-.058.175-.135.214Z"
              fill="#fff"
            />
          </svg>
        </a>
      </div>
    </footer>
  </main>
</div>
`;
